### DISCUSSION TITLE
live validation

### DISCUSSION THREAD

**Discussion by mlarcher** [mantinedev/mantine#1610] [Q&A] [answered]
Hi there,

I stumbled on this lib and find it really awesome ! I love how it seems to span from component library to layout and include form handling, always keeping it simple and efficient. Seriously, I think I haven't been that impressed since discovering jQuery in 2006 👍 

Anyhow, I have a question regarding useForm : is there a way to have live validation (i.e. onChange) ? It seems to me that errors are  only sent on submit, but it feels like a simple feature to implement (even though it should be opt-in for performance concerns) so I have to ask. 

Anyone can confirm that it's not currently possible or point me to the right direction ? Or eventually tell me if that's something on the roadmap or something that goes against some team principles ? Any information would be welcome as you can see :)

**Comment by rtivital** [comment, marked as answer]
Hi Matthieu, thanks for kind words! We will be adding an extra option for live validation in 5.0.

**Reply by mlarcher** [reply]
> Hi Matthieu, thanks for kind words! We will be adding an extra option for live validation in 5.0.

That's great news 💪
Do you have any idea when it will land?

**Reply by rtivital** [reply]
My best bet is August - September

**Reply by mlarcher** [reply]
Thanks for the quick reply 👌
